cheerio-scraper/src/crawler_setup.js

```diff
@@ -267,19 +258,14 @@ class CrawlerSetup {
                 globalStore: this.globalStore,
                 requestQueue: this.requestQueue,
                 customData: this.input.customData,
-                useRequestQueue: this.input.useRequestQueue !== false,
             },
             pageFunctionArguments: {
-                $,
+                ...crawlingContext,
                 get html() {
                     log.deprecated('Cheerio Scraper: Parameter context.html is deprecated use context.body instead.');
                     return contentType.type === 'text/html' ? body : null;
                 },
-                body,
-                json,
-                autoscaledPool,
-                request,
-                contentType,
+                autoscaledPool: crawler.autoscaledPool,
```

Comment:
when we expose crawler (from `crawlingContext`), we don't need to expose `autoscaledPool` anymore.